src/Forms/Form.php

```diff
@@ -297,7 +299,7 @@ public function __construct(
         $this->setName($name);
 
         // Form validation
-        $this->validator = ($validator) ? $validator : new RequiredFields();
+        $this->validator = ($validator) ? $validator : new RequiredFieldsValidator();
```

Comment:
We shouldn't be adding a `RequiredFieldsValidator` by default anymore. Add a `CompositeValidator` or nothing.

That was the AC that you crossed out, but I think it's important to retain. `RequiredFieldsValidator` is not an appropriate default.

Reply:
I've spilt the issue into 3 lots of PRs, this is the "rename classes" set of PRs and does not deal with "no longer call FormField->validate() within RequiredFields. That work is being done in the third lots of PRs.

Reply:
I'm not talking about no longer calling `FormField->validate()` within `RequiredFieldsValidator`. I'm talking about 
> - Anywhere that just adds RequiredFields to a form should instead add CompositeValidator with both a FormFieldsValidator and (if necessary) RequiredFieldsValidator validator
>    - Form::__construct() and SilverStripe\AssetAdmin\Forms\RemoteFileFormFactory::getForm() should explicitly exclude the RequiredFieldsValidator validator

This was crossed out but shouldn't have been imo - only the parts about `FormFieldsValidator` in the above should have been removed, not the parts about which validator(s) to include by default.

Reply:
If that's _also_ going to be included in the 3rd set of PRs that's fine, but right now there's no indication of that.

Reply:
The bottom 4 ACs on the issue will be dealt with with the third lot of PRs. These PRs only deal with the second AC ` RequiredFields should be renamed RequiredFieldsValidator to match the naming convention of other validators`

Reply:
I'm not talking about the bottom four ACs. Please read through my comments again - if you're still thinking I'm talking about those bottom ACs after that please DM me so we can figure out where the confusion is and get on the same page.

Reply:
I've added another AC "Anywhere that previously added RequiredFields to a form just to get FormField validation to work no longer adds a validator". This will be implemented in the 3rd lot of prs.